torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -757,6 +757,20 @@ def trace_triton_kernel_wrapper(
         proxy_args,
         name=func_overload.__name__ + "_proxy",
     )
+    kernel = kernel_side_table.get_kernel(proxy_args["kernel_idx"])
+    from torch._inductor.codegen.wrapper import (
+        user_defined_triton_kernel_transitive_closure_source_code,
+    )
+
+    kernel_source = user_defined_triton_kernel_transitive_closure_source_code(kernel)
+    constant_args = kernel_side_table.get_constant_args(proxy_args["constant_args_idx"])
```

Comment:
We should change how the constant args works (to also benefit the triton kernel export work). Constant args should be inputs to the node, unless they are non-fx-graphable types. The non-fx-graphable types should go into the side table.

Reply:
I guess I didnt want constant args to be spit between on graph and on side table, wanted to keep them all together originally

Reply:
We already split the constant args to do triton kernel non-strict export, because non-strict export doesn't have the concept of "constant args"